.github/workflows/build-linux.yml

```
@@ -0,0 +1,196 @@
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: aarch64-unknown-linux-gnu
+
+      - name: Install Linux ARM64 cross-compilation tools
+        run: |
+          sudo apt-get update
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)